bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/internal/CommonWidgetsDPIChangeHandlers.java

```diff
@@ -43,4 +45,12 @@ private static void handleItemDPIChange(Widget widget, int newZoom, float scalin
 			item.setImage(image);
 		}
 	}
-}
+	private static void handleCComboDPIChange(Widget widget, int newZoom, float scalingFactor) {
+		if (!(widget instanceof CCombo)) {
+			return;
+		}
+		CCombo combo = (CCombo) widget;
```

Comment:
```suggestion
		if (!(widget instanceof CCombo combo)) {
			return;
		}
```
a bit shorter variant